src/App.js

```diff
@@ -1,25 +1,176 @@
-import logo from './logo.svg';
 import './App.css';
```

Comment:
I've seen a couple of students putting all of their components in one file. Though this isn't a 'wrong' way to do this, I would discourage you from keeping them all in one file. Single Responsibility principle would be one reason, but also will make it less confusing to organize yourself when implementing state later. Again, not a wrong way, but really thing it makes sense to structure this in different files.